### ISSUE TITLE
Add aria-label attribute to IconButtons

### ISSUE THREAD

**Issue by yacdaniel** [craftercms/craftercms#7951]
### Duplicates

- [x] I have searched the existing issues

### Latest version

- [x] The issue is in the develop branch
- [ ] The issue is in the latest released 4.3
- [ ] The issue is in the latest released 4.2
- [ ] The issue is in the latest released 4.1
- [ ] The issue is in an older release (specified in the ticket)

### Describe the issue

Most of the `IconButtons` elements are missing the `aria-label` attribute. 

### Steps to reproduce

Steps:

1. Inspect with devtools the IconButtons in Studio UI (or use a tool like https://chromewebstore.google.com/detail/gdnpkbipbholkoaggmlblpbmgemddbgb?utm_source=item-share-cb) .
2. See the issue

### Relevant log output

```shell

```

### Screenshots and/or videos

![Image](https://github.com/user-attachments/assets/de1d2706-efde-4166-9d3e-8421c2dc0d8e)

**Comment by rart** [comment]
@jvega190 please review and adjust aria-labels and titles for all status icons on dashboard, legend, item display, workflow states and other places where icons convey meaning.

**Comment by yacdaniel** [comment]
Verified fixed in `v5.0.0-SNAPSHOT`
